kubernetes/client/apis/core_v1_api.py

```diff
@@ -1007,7 +1007,7 @@ def connect_get_namespaced_pod_exec_with_http_info(self, name, namespace, **kwar
                                         auth_settings=auth_settings,
                                         callback=params.get('callback'),
                                         _return_http_data_only=params.get('_return_http_data_only'),
-                                        _preload_content=params.get('_preload_content', True),
```

Comment:
the _preload_content tries to coerce the response into a json. would be great to find another spot to put this.